srcpkgs/v2raya/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'v2raya'
+pkgname=v2raya
+version=1.5.9.1698.1
+revision=1
+wrksrc="v2rayA-${version}"
+build_style="go"
+go_import_path="github.com/v2rayA/v2rayA"
+go_ldflags='-X github.com/v2rayA/v2rayA/conf.Version=unstable-'"$version"' -s -w'
+hostmakedepends="go nodejs yarn"
+depends="v2ray>=4.37.0-1"
+short_desc="Web GUI client of Project V which supports various protocals"
+maintainer="ketlrznt <tansuanyinliao8888@gmail.com>"
+license="AGPL-3.0"
+homepage="https://github.com/v2rayA/v2rayA"
+distfiles="https://github.com/v2rayA/v2rayA/archive/v${version}.tar.gz"
+checksum="247a357230c616bf48309c61d119686e4ad56939c05afef584c45051e9dc6220"
+
+export GO11MODULE=on
+
+do_build() {
+	cd gui
+	yarn --check-files
+	OUTPUT_DIR="${wrksrc}/service/server/router/web" yarn build
+	cd ../service
+	go build -ldflags="$go_ldflags" -o "${GOPATH}/bin/v2raya"
+}
+
+do_install() {
+	# binary
+	vbin ${GOPATH}/bin/v2raya
+	# desktop entry
+	vinstall install/universal/v2raya.desktop 0644 usr/share/applications
+	# configurationo dir
+	vmkdir etc/$pkgname 0750
```

Comment:
use the make_dirs and conf_files for this, empty dirs are not included in the pkg automatically

Reply:
Thanks. I would like to remove this as v2raya configuration file location is set in v2raya service.

Reply:
done.